pandas/core/generic.py

```diff
@@ -2211,8 +2211,9 @@ def to_excel(
             via the options ``io.excel.xlsx.writer`` or
             ``io.excel.xlsm.writer``.
 
-        merge_cells : bool, default True
-            Write MultiIndex and Hierarchical Rows as merged cells.
+        merge_cells : bool or 'columns', default False
+            Write MultiIndex and Hierarchical Rows as merged cells if True.
```

Comment:
Now that we're adding in `columns` as an option, I think this would be clearer if it said `Write MultiIndex index and columns as merged cells...`.

Reply:
Sure!